spotipy2/methods/playlist.py

```diff
@@ -0,0 +1,46 @@
+from __future__ import annotations
+from typing import List, Optional
+
+import spotipy2
+from spotipy2.types import Playlist, Track
+
+
+class PlaylistMethods:
+    async def get_playlist(
+        self: spotipy2.Spotify, playlist_id: str  # type: ignore
+    ) -> Playlist:
+        return Playlist.from_dict(
+            await self._get(f"playlists/{self.get_id(playlist_id)}")
+        )
+
+    async def get_playlist_tracks(
+        self: spotipy2.Spotify,  # type: ignore
+        playlist_id: str,
+        market: Optional[str] = None,
+        limit: Optional[int] = None,
+        offset: Optional[int] = None,
+    ) -> List[Track]:
+        limit = limit or 20
```

Comment:
Remove the default limit in `get_playlist_tracks`. The philosophy is to let Spotify decide the default value (Also shouldn't be better to set `limit: int = 20` then?).

Reply:
if `None` get's passed spotify will default to 20

![image](https://user-images.githubusercontent.com/42355410/157292554-ca2debc4-fa46-45db-a8a4-e08369e66fad.png)

> https://developer.spotify.com/documentation/web-api/reference/#/operations/get-playlists-tracks

>  (Also shouldn't be better to set limit: int = 20 then?)
yeah, it would be